core/src/main/java/com/novoda/noplayer/exoplayer/EventListener.java

```diff
@@ -0,0 +1,58 @@
+package com.novoda.noplayer.exoplayer;
+
+import com.google.android.exoplayer2.ExoPlaybackException;
+import com.google.android.exoplayer2.ExoPlayer;
+import com.google.android.exoplayer2.PlaybackParameters;
+import com.google.android.exoplayer2.Timeline;
+import com.google.android.exoplayer2.source.TrackGroupArray;
+import com.google.android.exoplayer2.trackselection.TrackSelectionArray;
+
+import java.util.List;
+
+class EventListener implements ExoPlayer.EventListener {
```

Comment:
This is a bit funny actually 😄  so this listens to `exoPlayer` events and passes them to the forwarders. I guess we could call it a forwarder too but it's more important that it listens to events... I guess 🤷‍♂️